src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = new String[]

```

Comment:
```suggestion
    private static final String[] colors = new String[]

```
Неймінг капсом потрібно писати лише для immutable класі або примітивів

Reply:
дякую за науку, поради та пропозиції, запам"ятаю та намагатимусь враховувати у майбутньому!